src/App.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  AppBar,
+  Alert,
+  Box,
+  Button,
+  Container,
+  FormGroup,
```

Comment:
nit: The linter complains that this import is unused so maybe we should remove it.

Reply:
Hmm, but we use all of them but `FormGroup`. I removed that now

Reply:
@Dominik1999 Ah sorry, I should have clarified. Github automatically marks additional 3 lines above (probably to make locating the code easier). I did mean only `FormGroup`.